cypress/integration/autocomplete.spec.ts

```diff
@@ -6,7 +6,7 @@ describe('Autocomplete in the editor', () => {
     // Wait for the autocomplete to show up
     cy.contains('assign');
 
-    cy.getMonacoEditor().type('{downarrow}{downarrow}{enter}');
+    cy.getMonacoEditor().type('{downarrow}{enter}');
```

Comment:
Now we only have one `assign` import in the suggestions, so one down arrow is enough.